app/mailers/user_notifications.rb

```diff
@@ -153,27 +153,45 @@ def account_silenced(user, opts = nil)
 
     return unless user_history = opts[:user_history]
 
-    build_email(
-      user.email,
-      template: "user_notifications.account_silenced",
-      locale: user_locale(user),
-      reason: user_history.details,
-      silenced_till: I18n.l(user.silenced_till, format: :long)
-    )
+    if user.silenced_till > DateTime.now + 500.years
```

Comment:
Can we instead have a method on the user model like `suspended_forever?` that we can use everywhere?

Reply:
That'll be prefect 👍, done.